### PR TITLE
Fixed: UI Glitch - on Secure Submit payment page

### PR DESCRIPTION
**Pull request by mayur-vachchewar**

### CODE REVIEW

**Comment by mayur-vachchewar** [comment]
@MSmedal could you please review and approve this PR.

File: readme.txt
Comment (MSmedal) [inline readme.txt:1]: Are you testing on the latest version of WP? If so, please update line 4 also. I tested using 6.5.5 and it seems to work well

**Review by MSmedal — CHANGES_REQUESTED** [review]
All seems well! Please advise on the single housekeeping item I raised and we'll go from there.

File: readme.txt
Comment (mayur-vachchewar) [reply readme.txt:1]: @MSmedal yes, it tested upto 6.5.5, updated the tested upto version as well.
For you reference: 
image

**Review by MSmedal — APPROVED** [review]

**Pull request merged**